core-services/release-controller/admin_deploy-ocp-controller-s390x.yaml

```diff
@@ -147,3 +147,25 @@ items:
   metadata:
     name: release-upgrade-graph
     namespace: ocp-s390x
+
+# Place holder for ci-release/files-pull-secret
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: files-pull-secret
+    namespace: ci-release-s390x
+
+# Place holder for ci-release/git-credentials
```

Comment:
You don't really need these

Reply:
The Secret Mirroring instructions imply that they need to exist prior to performing the mirroring.  The secrets will be created if they dont exist?

Reply:
Is this the source or the target? The target (where we mirror to) will be created. The source (where we mirror from) should be recorded in the BitWarden script.

Reply:
They are targets.  I've removed these changes.